.env.example

```diff
@@ -0,0 +1,3 @@
+DOCKER_TAG=latest
+OPENID_CLIENT_ID=<ID>
+OPENID_CLIENT_SECRET=<SECRET>
```

Comment:
So env files with docker-compose are kinda jank, but I _think_ we can leave the existing `.env` file, and use a separate one for the secrets. Can you try putting the original `.env` back with just the `DOCKER_TAG`, then add a file like `api/vars.env`, and under the api container in the `docker-compose.yml`, you set `env_file: api/vars.env`? I think it'll load both and we can add `api/vars.env` to the gitignore. Then have like `api/vars.example.env` with placeholder/empty values?

Reply:
Now that i think of it, we probaly should just have a yaml/json file that is a map like
```yaml
<provider name>:
       client_id: <id>
       client_secret: <secret>
       issuer_url: <url>
```

That way we dont need to make new env vars for each provider